pkg/kubeapiserver/admission/initializer.go

```diff
@@ -73,14 +81,18 @@ type pluginInitializer struct {
 var _ admission.PluginInitializer = pluginInitializer{}
 
 // NewPluginInitializer constructs new instance of PluginInitializer
-func NewPluginInitializer(internalClient internalclientset.Interface,
+func NewPluginInitializer(
+	internalClient internalclientset.Interface,
+	externalClient clientset.Interface,
```

Comment:
I prefer the plumbing mechanism I did in my PR--continuing to add more parameters here gets crazy.